.github/workflows/main.yml

```
@@ -64,12 +66,12 @@ jobs:
         with:
           registry-url: 'https://registry.npmjs.org'
           node-version-file: '.nvmrc'
+          cache: 'npm'
 
       - name: Install
-        run: npm install --no-save --no-fund --no-audit --legacy-peer-deps
+        run: npm ci --no-save --no-fund --no-audit --legacy-peer-deps
```

```suggestion
        run: npm ci
```

these are `npm install` flags, not `npm ci`'s, right? ` --no-save --no-fund --no-audit --legacy-peer-deps`

---

They are also valid for `ci`: https://docs.npmjs.com/cli/v10/commands/npm-ci#configuration

---

Yes, we need them. We can also add them in the `npmrc`